frontend/packages/component-library/src/dropdown/customDropdown.module.scss

```
@@ -80,7 +80,7 @@
   .dropdownMenuContainer {
     position: absolute;
     bottom: auto;
-    z-index: 1000;
+    z-index: variables.$zindex-modal-backdrop;
```

nit: looks like this should be vice versa, 

```suggestion
    z-index: variables.$zindex-dropdown;
```